### PR TITLE
Added README - apigee

### PR DESCRIPTION
**Pull request by rajeevr2715**

### CODE REVIEW

File: feature-extensions/apigee/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+## Summary
+- **Make sure you have an Apigee account with at least two environments**
+- **This installer will use whatever AWS credentials you have configured by running 'aws configure'**
+- **Please make sure you are using the same AWS credentials you used to install your Jazz deployment**
+
+## USAGE
+usage: setup.py [-h] [--aws-region AWS_REGION]
```
Comment (devsatishm) [inline feature-extensions/apigee/README.md]: You can try enclosing your sh command using ``` for better formatting.

File: feature-extensions/apigee/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                [--jazz-stackprefix JAZZ_STACKPREFIX] [--scm-repo SCM_REPO]
+                [--scm-username SCM_USERNAME] [--scm-password SCM_PASSWORD]
+                [--scm-pathext SCM_PATHEXT] [--jenkins-url JENKINS_URL]
+                [--jenkins-username JENKINS_USERNAME]
+                [--jenkins-password JENKINS_PASSWORD]
+                {install,uninstall} ... apigee_host apigee_org apigee_prod_env
```
Comment (devsatishm) [inline feature-extensions/apigee/README.md]: Can you  complete  this for other arguments like `apigee_host`, `apigee_org` etc. ?

File: feature-extensions/apigee/README.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                apigee_dev_env apigee_svc_prod_host apigee_svc_dev_host
+                apigee_username apigee_password
+
+
+- **Even though the Install/Uninstall is possible with Parameterized approach, it will ask
```
Comment (devsatishm) [inline feature-extensions/apigee/README.md]: Add this as a note, remove bold.

File: feature-extensions/apigee/README.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    user raw inputs for the missed parameters**
+
+#### INSTALL #####
```
Comment (devsatishm) [inline feature-extensions/apigee/README.md]: =>  ## Install

=>  ### With all the parameters
.... 
=>  ### With missing parameters
....

=>  ## Uninstall

=>  ### With all the parameters
.... 
=>  ### With missing parameters
....

File: feature-extensions/apigee/README.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+
+## With all the parameters as argument
+**Follow the below steps**
+1. cd JAZZ_INSTALLER_DIRECTORY/feature-extensions/apigee
+2. python setup.py --aws-region REGION --jazz-stackprefix STACKPREFIX --scm-repo SCMREPO --scm-username SCMUSERNAME --scm-password SCMPASSWORD --scm-pathext / --jenkins-url JENKINSURL --jenkins-username JENKINSUSER --jenkins-password JENKINSPASSWORD install https://api.enterprise.apigee.com APIGEEORG PRODENV DEVENV APIGEE_PROD_SERVICEHOST APIGEE_DEV_SERVICEHOST APIGEEUSERNAME APIGEEPASSWORD
```
Comment (devsatishm) [inline feature-extensions/apigee/README.md]: Enclosing your sh commands within ```sh  for better formatting.

File: feature-extensions/apigee/README.md (hunk 2 above)
Comment (rajeevr2715) [reply feature-extensions/apigee/README.md]: All the arguments has been given, like 
python setup.py [ARGUMENTS] {install,uninstall} [APIGEE ARGUMENTS]

File: feature-extensions/apigee/README.md (hunk 3 above)
Comment (rajeevr2715) [reply feature-extensions/apigee/README.md]: Done

File: feature-extensions/apigee/README.md (hunk 1 above)
Comment (rajeevr2715) [reply feature-extensions/apigee/README.md]: Done

File: feature-extensions/apigee/README.md (hunk 4 above)
Comment (rajeevr2715) [reply feature-extensions/apigee/README.md]: Done

File: feature-extensions/apigee/README.md (hunk 5 above)
Comment (rajeevr2715) [reply feature-extensions/apigee/README.md]: Done

**Pull request merged**